alws/alembic/versions/46393610f521_rbac_for_signing_and_releasing.py

```diff
@@ -33,11 +33,13 @@ def upgrade():
         sa.PrimaryKeyConstraint('platform_id', 'role_id')
     )
     op.add_column('build_releases', sa.Column('team_id', sa.Integer(), nullable=True))
+    op.create_foreign_key('build_releases_team_id_fkey', 'build_releases', 'teams', ['team_id'], ['id'])
     # ### end Alembic commands ###
 
 
 def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint('build_releases_team_id_fkey', 'build_releases', type_='foreignkey')
```

Comment:
can we insert these changes in new migration instead?

Reply:
Moved